storybook/src/components/Dialog/Dialog.stories.tsx

```diff
@@ -5,26 +5,29 @@
 
 import { Button, Dialog, Heading, Paragraph } from '@amsterdam/design-system-react/src'
 import { Meta, StoryObj } from '@storybook/react'
+import { MouseEvent } from 'react'
+
+const closeDialog = (event: MouseEvent<HTMLButtonElement>) => event.currentTarget.closest('dialog')?.close()
```

Comment:
For these long one-liners I'd prefer curly braces and the return keyword, but I don't if our linter/prettier allows that.